samza-rest/src/main/java/org/apache/samza/rest/model/ApplicationInfoWrapper.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.samza.rest.model;
+
+import org.codehaus.jackson.annotate.JsonProperty;
+import org.codehaus.jackson.map.annotate.JsonRootName;
+import java.util.List;
+import java.util.Objects;
+
+
+@JsonRootName("apps")
+public class ApplicationInfoWrapper {
+  @JsonProperty("app")
```

Comment:
Misleading name, should be "apps" since the value is a list.
Even better would be to make this a "application ID" -> ApplicationInfo Map so that we can look up by the id directly (whatever that is).

Reply:
its unfortunate, but "app" is the name of the LIST of apps that is returned from the API. https://hadoop.apache.org/docs/stable/hadoop-yarn/hadoop-yarn-site/ResourceManagerRest.html#Cluster_Applications_API

Since the returned JSON data is a list, i'm not sure if making this a Map will work for jackson ser/deser.